### PR TITLE
fix(torch_api.distributed): cycle dependence

### PR DESCRIPTION
**Pull request by shjwudp**

### CODE REVIEW

File: bagua/service/autotune_service.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def record_autotune_log(
             }
         )
         cols = OrderedDict(cols)
-        logging.info("cols={}".format(cols))
+        logging.debug("cols={}".format(cols))
```
Comment (NOBLES5E) [inline bagua/service/autotune_service.py:117]: should be in separate PR

File: bagua/torch_api/communication.py [hunk 2]
```diff
@@ -33,26 +32,25 @@ def is_initialized():
     return _global_state is not None
 
 
-def init_process_group(init_method: str = "dist://", device_id=None):
```
Comment (NOBLES5E) [inline bagua/torch_api/communication.py:36]: should be in separate pull requests.

File: bagua/torch_api/algorithms/distributed_module.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        super(DistributedModule, self).__init__()
+        self.module = module
+        if hasattr(module, "_bagua_params_and_buffers_to_ignore"):
+            self.parameters_to_ignore = [
+                ("module." + k) for k in module._bagua_params_and_buffers_to_ignore  # type: ignore
+            ]
+        else:
+            self.parameters_to_ignore = []
+
+    def unwarp(self):
```
Comment (NOBLES5E) [inline bagua/torch_api/algorithms/distributed_module.py:22]: `unwrap`

File: bagua/torch_api/distributed.py [hunk 4]
```diff
@@ -29,7 +29,8 @@
 from .algorithms.decentralize import DecentralizedReducer
 from .algorithms.allreduce import Allreducer, ScatterGatherAllreducer
 from .compression import Compressor
-from bagua.service.autotune_service import BaguaHyperparameter
+import traceback
```
Comment (NOBLES5E) [inline bagua/torch_api/distributed.py:32]: does not seem to be used

Comment (shjwudp) [reply bagua/torch_api/distributed.py]: Used for torch hook debugging information

File: bagua/torch_api/algorithms/distributed_module.py (hunk 3 above)
Comment (shjwudp) [reply bagua/torch_api/algorithms/distributed_module.py]: DONE
